### DISCUSSION TITLE
Software development

### DISCUSSION THREAD

**Discussion by Ad3oluwano1** [community/community#73454] [New to GitHub]
### Body

How do you start a new GitHub

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by RumethSandinu** [comment]
Question is not understandable

**Comment by Akash1134** [comment]
👋 Welcome to the GitHub Community on Discussions, @Ad3oluwano1 ! Happy to have you here 🙂

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776).

**Comment by har8shith** [comment]
hello everyone am very new to GitHub just learning how to use it efficiently happy to connect : )

**Reply by prudhvisai26** [reply]
Hey there, if you want to know about GitHub go through the documentation or you can go to the youtube channel called freecodecamp.
They teach you from basics that may help you.
link to the yt channel: [github](https://www.youtube.com/watch?v=RGOj5yH7evk&ab_channel=freeCodeCamp.org)
Happy Coding...
